discovery/refresh/refresh.go

```diff
@@ -28,6 +28,7 @@ import (
 type Options struct {
 	Logger              *slog.Logger
 	Mech                string
+	Config              string
```

Comment:
`Config` suggests to me that it's a struct.
```suggestion
	ConfigName          string
```

Reply:
Switched to SetName: https://github.com/prometheus/prometheus/commit/00f7ed868433b995d06e31c6f04be9fd0afed82d